content/post/setup-kubernetes-raspberry-pi-cluster.md

```diff
@@ -70,106 +70,121 @@ The installation requries root privileges. Retrieve them by
 
+```console
+$ curl -s https://packages.cloud.google.com/apt/doc/apt-key.gpg | apt-key add -
+$ echo "deb http://apt.kubernetes.io/ kubernetes-xenial main" > /etc/apt/sources.list.d/kubernetes.list
 ```
-curl -s https://gist.githubusercontent.com/MathiasRenner/e74f91e23b987424f36a5e61acd4f3e8/raw/1001f8a1b4ad43aac3b3b1b15cc9b8dd5c1616c4/install.sh | bash
+
+Then just install `kubeadm` on every node:
+```console
+$ apt-get update && apt-get install -y kubeadm
 ```
 
-After the previous command has been finished, initialize K8s on the master node with
+After the previous command has been finished, initialize Kubernetes on the **master node** with
 
+```console
+$ kubeadm init --pod-network-cidr 10.244.0.0/16
 ```
-kubeadm init --pod-network-cidr 10.244.0.0/16
-```
-It is important that you add the `--pod-network-cidr` command as given here. //TODO why?
-
-If you are connected via WIFI instead of Ethernet, add `--api-advertise-addresses=<ip-address>` as parameter to  `kubeadm init` in order to publish Kubernetes' API via WiFi.
+It is important that you add the `--pod-network-cidr` command as given here, **if you're using flannel** as in this example here.
+That's because flannel is storing it's metadata about Pod CIDR allocations, and therefore we need to tell _Kubernetes_ first which subnet we want to use.
 
-After K8s has been initialized, the last lines of your terminal should look like this:
+If you are connected via WIFI instead of Ethernet, add `--api-advertise-addresses=<wifi-ip-address>` as parameter to `kubeadm init` in order to publish Kubernetes' API via WiFi.
+Also feel free to explore the other options that exist for `kubeadm init`.
 
+After Kubernetes has been initialized, the last lines of your terminal should look like this:
 
 ![init](/images/kubernetes-setup-cluster/init.png)
 
 
-Next, as told by that output, let all other nodes join the cluster via the given `kubeadm join` command. In this case, it is:
-```
-kubeadm join --token=bb14ca.e8bbbedf40c58788 192.168.0.34
+Next, as told by that output, let all other nodes join the cluster via the given `kubeadm join` command. It will look something like:
+```console
+$ kubeadm join --token=bb14ca.e8bbbedf40c58788 192.168.0.34
 ```
 
-After some seconds, you should see all nodes in your cluster when executing the following on the master node:
-```
-kubectl get nodes
+After some seconds, you should see all nodes in your cluster when executing the following _on the master node_:
+```console
+$ kubectl get nodes
 ```
 
 ![k8S](/images/kubernetes-setup-cluster/get-nodes.png)
 
-Finally, we need to setup flannel as the network driver. Meanwhile, [Weave](https://www.weave.works/products/weave-net/) is also available, but here we just present flannel:
-```
-curl -sSL https://rawgit.com/coreos/flannel/master/Documentation/kube-flannel.yml | sed "s/amd64/arm/g" | kubectl create -f -
+Finally, **we need to setup the Pod network driver** from the master node. 
```

Comment:
Let's change the style to a more instructions-like one and remove the "from the master node" here - and just add a "Run this **on the master node**" just before the command.